Makefile.standalone

```diff
@@ -0,0 +1,52 @@
+
+## path configuration (FHS-style)
+PREFIX ?= /usr
+BINDIR ?= $(PREFIX)/bin
+SBINDIR ?= $(PREFIX)/sbin
+
+## toolchain commands
+export GO ?= go
+export INSTALL ?= install
+export LN_S ?= ln -s
+export MKDIR_P ?= mkdir -p
+
+BUILD_TMP ?= $(CURDIR)/tmp
+BUILD_OUT ?= $(BUILD_TMP)/build
+export GOCACHE ?= $(BUILD_TMP)/go-cache
+export LDFLAGS=-s -w
+
+BUILD_GO_ARCH := $(shell $(GO) env GOARCH | sed -e 's~^386~i386~;')
+BUILD_GO_OS := $(shell $(GO) env GOOS)
+
+EXEC_NAME=dockerd-$(BUILD_GO_OS)-$(BUILD_GO_ARCH)
+EXEC_OUT=$(BUILD_OUT)/$(EXEC_NAME)
+
+# make sure it fails ...
+export GOPATH=/proc/gopath
```

Comment:
?????

Reply:
> ?????

This is an ugly workaround for preventing go from trying any updates of vendored stuff (which will be written to gopath) and thus completely break things. This silly path is (almost) guaranteed to be ro even when being root within a container.

Unfortunately could not find any better way to force go into just using what's there w/o ever attempting to download anything.